cmd/crowdsec-cli/explain.go

```diff
@@ -108,3 +112,16 @@ cscli explain --dsn "file://myfile.log" --type nginx
 
 	return cmdExplain
 }
+
+func getLineCountForFile(filepath string) int {
```

Comment:
might need a defer'ed close ? might bail out after X lines anyway (to avoid losing time here)

Reply:
thx, fixed now